packages/dev/gui/src/2D/controls/image.ts

```diff
@@ -583,10 +583,12 @@ export class Image extends Control {
         this._domImage = engine.createCanvasImage();
         // need to add to enforce rendering
         const imgElement = this._domImage as HTMLImageElement;
-        if (imgElement.style) {
+        let addedToDom = false;
+        if (imgElement.style && this._source?.endsWith(".svg")) {
             imgElement.style.visibility = "hidden";
             imgElement.style.position = "absolute";

```

Comment:
this needs to have something like "top: 0" or this will change the page layout and increase height of parent container.

also quality check: ensure these images are not appearing overtop of the canvas. Prepend them to the parent container to ensure they are before the canvas/3d view. Because of native z layering which is determined by ascending order (afaik, from past experience), that will cause them to always be underneath the canvas.

Reply:
concern is that there'd be a scrollbar appearing briefly due to this. Not 100% sure if it would be observed by the user, but its highly likely if the onload is an async callback with time to render the page in between.
A screenshot regression test will not reveal the temporary layout change if its only taken at the end. A video would of course.

Reply:
the add and remove happen on the same frame, synchronous. This should not change user layout